docs: update the quickstart guide (#2208)
bentoml/BentoML: docs/source/quickstart.rst

Save Models
-----------

> We begin by with saving a trained model instance to BentoML's local

Reviewer: ```suggestion
We begin by saving a trained model instance to BentoML's local
```

:ref:`model store <bento-management-page>`.

> If models you wish to use are already saved to disk, they can also be added to BentoML with the

Reviewer: ```suggestion
If the model are already saved to disk, they can also be added to BentoML with the
```

    clf.fit(X, y)

    bentoml.sklearn.save("iris_clf", clf)
    # [08:34:16 AM] INFO     Successfully saved Model(tag="iris_clf:svcryrt5xgafweb5",
    #                        path="/home/user/bentoml/models/iris_clf/svcryrt5xgafweb5/")
    # Tag(name='iris_clf', version='svcryrt5xgafweb5')

> The :ref:`ML framework specific API <frameworks-page>`, `bentoml.sklearn.save()`, will save the Iris

Reviewer: ```suggestion
The :ref:`ML framework specific API <frameworks-page>`, :code:`bentoml.sklearn.save()`, will save the Iris
```

> Classifier to a local model store managed by BentoML. You can then load the runner for this model

Reviewer: ```suggestion
Classifier to a local model store managed by BentoML. Users can then load the runner for this model
```

Author: I'd say 2nd person is quite common in docs, so I think this is fine.

with the `load_runner()` API.

.. code-block:: python

    iris_clf_runner = bentoml.sklearn.load_runner("iris_clf:latest")
    iris_clf_runner.run(np.array([5.9, 3. , 5.1, 1.8]))

> Models can also be managed via the `bentoml models` CLI command. For more information use

Reviewer: ```suggestion
Models can also be managed via :code:`bentoml models` CLI command. For more information use
```

Author: ```suggestion
Models can also be managed via the :code:`bentoml models` CLI command. For more information use
```

> `bentoml models --help`.

Reviewer: ```suggestion
:code:`bentoml models --help`.
```

        # Define post-processing logic
        return result

> In this example, we defined `numpy.ndarray` as the input and output type. More options such as

Reviewer: ```suggestion
In this example, we defined a :code:`numpy.ndarray` as the input and output type. More options such as
```

Author: ```suggestion
In this example, we defined :code:`numpy.ndarray` as the input and output type. More options such as
```

> `pandas.dataframe` and `PIL.image` are also supported IO types. To see all supported options, see

Reviewer: ```suggestion
:code:`pandas.DataFrame` and :code:`PIL.image` are also supported IO types. To see all supported options, see
```

@API and IO Descriptors.

We now have everything we need to serve our first request. Launch the server in debug mode by

> running the `bentoml serve` command in the current working directory. Using the `--reload` option

Reviewer: ```suggestion
running the :code:`bentoml serve` command in the current working directory. Using the `--reload` option
```

> allows the server to reflect any changes made to the `bento.py` module without restarting the

Reviewer: ```suggestion
allows the server to reflect any changes made to the `bento.py` module without the need to restart the
```

> server.

Reviewer: ```suggestion

```

> allows the server to reflect any changes made to the `bento.py` module without restarting the
> server.

Author: ```suggestion
allows the server to reflect any changes made to the `bento.py` module without restarting:
```

    [10:18:42 AM] INFO     Will watch for changes in these directories: ['/home/user/devel/bentoml-quickstart']                                                              config.py:334
                  INFO     Uvicorn running on http://127.0.0.1:5000 (Press CTRL+C to quit)                                                                                   config.py:554
                  INFO     Started reloader process [97796] using statreload                                                                                              basereload.py:56
    [10:18:43 AM] INFO     Started server process [97808]                                                                                                                     server.py:84
                  INFO     Waiting for application startup.                                                                                                                       on.py:45
                  INFO     Application startup complete.                                                                                                                          on.py:59

> We can then send requests to the newly started service with any HTTP client.

Reviewer: ```suggestion
We can then send requests to the newly started service with any HTTP client:
```

        requests.post(
            "http://127.0.0.1:5000/predict",
            headers={"content-type": "application/json"},
            data="[5,4,3,2]").text

    .. code-tab:: bash

>         > curl \

Reviewer: This should be in oneline so that the sphinx doesn't format the escaped space in the rendered code block

Author: I don't see what you mean; the code block looks fine?

    [INFO] Serving BentoML Service "iris_classifier_service"
    [INFO] API Server running on http://0.0.0.0:5000

Lastly, we can :ref:`containerize bentos as Docker images <containerize-bentos-page>` using the

> `bentoml container` CLI command and manage bentos at scale using the

Reviewer: ```suggestion
:code:`bentoml container` CLI command and manage bentos at scale using the
```
